src/bundle/Resources/config/services/forms.yaml

```diff
@@ -18,6 +18,7 @@ services:
 
     EzSystems\EzPlatformAdminUi\Form\Type\:
         resource: '../../../lib/Form/Type'
+        exclude: '../../../lib/Form/Type/ContentType/{FieldDefinitionsCollectionType}.php'
```

Comment:
Looking into future code growth this is bad practice IMHO. Imagine there are 10 more classes. This would become unmaintainable and unreadable. But I guess it requires more internal discussion and decision, so +1 for now.

Reply:
@alongosz I'll create a follow up PR containing all service declarations, but it will require a more thorough testing.